platform/windows/README.md

```diff
@@ -83,7 +83,7 @@ If all went well and target `mbgl-render` or `ALL_BUILD` was chosen, there shoul
 
 This should produce an `out.png` map tile image with the default MapLibre styling from [the MapLibre demo](https://maplibre.org/).
 
-![Sample image of world from mbgl-render command](/misc/sample-maplibre-style-mbgl-render-out.png)
+![Sample image of world from mbgl-render command](https://github.com/user-attachments/assets/b1a9355f-731e-4a0f-ab90-68c5ae4beb65)
```

Comment:
I don't think this is a good path: repo's content should not depend on external services like this. When all content is together, cloning it is enough -- and we can easily do any kind of markdown book packaging to create a site. When it's external, it can't not be part of the docs.  You could do this as a separate PR and discuss it there of course.

Reply:
I reverted the change.